test_app/tests/rbac/api/test_user_permissions.py

```diff
@@ -197,6 +197,57 @@ def test_team_admins_can_add_children(self, user, user_api_client, organization,
         assert rando.has_obj_perm(inventory, 'change')
 
 
+@pytest.mark.django_db
+class TestRelatedUserListView:
+    def _initial_check(self, url, user_api_client, count=0):
+        response = user_api_client.get(url)
+        assert response.status_code == 200
+        assert response.data['count'] == count
+
+    def _assign_users(self, role_definition, object):
+        members = [
+            User.objects.create(username='rando1'),
+            User.objects.create(username='rando2'),
+            User.objects.create(username='rando3'),
+        ]
+        role_definition.give_permission(members[0], object)
+        role_definition.give_permission(members[1], object)
+
+    def test_org_admin_list_org_members(self, user, user_api_client, organization, org_member_rd, org_admin_rd):
+        org_admin_rd.give_permission(user, organization)
+
+        url = reverse('organization-users-list', kwargs={'pk': organization.pk})
+        self._initial_check(url, user_api_client)
+        self._assign_users(org_member_rd, organization)
+
+        response = user_api_client.get(url)
+        assert response.status_code == 200
+        assert response.data['count'] == 2
+
+    def test_org_member_list_org_members(self, user, user_api_client, organization, org_member_rd):
+        org_member_rd.give_permission(user, organization)
+
+        url = reverse('organization-users-list', kwargs={'pk': organization.pk})
+        self._initial_check(url, user_api_client, 1)
+        self._assign_users(org_member_rd, organization)
+
+        response = user_api_client.get(url)
+        assert response.status_code == 200
+        assert response.data['count'] == 3
+
+    def test_superadmin_list_org_members(self, user, user_api_client, organization, org_member_rd):
+        user.is_superuser = True
+        user.save()
```

Comment:
Would be more direct to use `admin_api_client` instead of doing this. But again, still perfectly functional, just a suggestion for code cleanliness.